File: tipedata.php
<?php
$nim = 'A';
$nama = "Nabill Handika Pratama";
$umur = 18;
$nilai = 82.25;
$status = TRUE;

echo "---Belajar Tipe Data --- <br><br>";
echo "Tipe Data Char :" .$nim."<br>";
echo "Tipe Data String :$nama<br>";
printf "Tipe Data Int :".$umur;
printf "<br>";
printf ("Tipe Data Float :%.5f<br><br>",$nilai);
echo "Tipe Data Boleean :<br><br>";
if($status)
    echo "Status : Aktif";
else 
    echo "Status : Tidak Aktif";
?>

File: tipedata.cpp
#include <stdio.h>
#include <conio.h>
#include <iostream>
using namespace std;

int main()
{
    char a[38] = "---BELAJAR TIPE DATA---";
    short b;
    int c;
    long d;
    float e;
    double f;
    int g = 1;
    int h = 2;

    b = 10000;
    c = 1000000;
    d = 1000000000;
    e = 3,12;
    f = 12345;

    cout << a << endl;
    cout << "tipe data short :" << b << endl;
    cout << "tipe data int :" << c << endl;
    cout << "tipe data long : " << d << endl;
    cout << "tipe data float : " << e << endl;
    cout << "tipe data souble : " << f << endl;
    cout << endl;

    cout << "Berikut Hasil Boolean : " << endl;
    cout << (g > h); // 
    cout << endl; 
    cout << (g < h); //
    cout << endl;
    cout << (g == h); //
    cout << endl;
    cout << (g != h); // 

    return 0;
}


